homework/index.js

```diff
@@ -30,13 +30,162 @@
+    const tbody = createAndAppend('tbody', table, { id: 'tbody' });
+
+    const defaultRepo = defaultSelectedRepository(data, select[select.selectedIndex].text);
+
+    function showSelectedConts(dataCont) {
+      const rightDiv = createAndAppend('div', container, {
+        id: 'rightDiv',
+        class: 'right-div whiteframe',
+      });
+      createAndAppend('p', rightDiv, {
+        class: 'contributor-header',
+        text: 'Contributions :',
+      });
+      const ul = createAndAppend('ul', rightDiv, { id: 'ulList', class: 'contributor-list' });
+      dataCont.forEach(element => {
+        const li = createAndAppend('li', ul, {
+          class: 'contributor-item',
+          arialabel: element.login,
+          tabindex: 0,
+        });
+        createAndAppend('img', li, {
+          src: element.avatar_url,
+          height: 48,
+          class: 'contributor-avatar',
+        });
+        const contInfoDiv = createAndAppend('div', li, { class: 'contributor-data' });
+        createAndAppend('div', contInfoDiv, { text: element.login });
+        createAndAppend('div', contInfoDiv, {
+          class: 'contributor-badge',
+          text: element.contributions,
+        });
+        li.addEventListener('click', () => {
+          window.open(element.html_url, '_blank');
+        });
+      });
+    }
+    fetchJSON(defaultRepo.contributors_url, (err, contributors) => {
+      if (err) {
+        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+      } else {
+        showSelectedConts(contributors);
+      }
+    });
+
+    function showSelectedRepo(selectedRepo) {
+      if (selectedRepo.name !== null) {
+        const tr1 = createAndAppend('tr', tbody);
+        createAndAppend('td', tr1, { class: 'label', text: 'Repository :' });
+        const td2 = createAndAppend('td', tr1);
+        const repoLink = createAndAppend('a', td2, { href: '', target: '_blank', text: '' });
+        repoLink.textContent = selectedRepo.name;
+        repoLink.href = selectedRepo.html_url;
+      }
+      if (selectedRepo.description !== null) {
+        const tr2 = createAndAppend('tr', tbody);
+        createAndAppend('td', tr2, { class: 'label', text: 'Description :' });
+        const repoDescription = createAndAppend('td', tr2, { text: '' });
+        repoDescription.textContent = selectedRepo.description;
+      }
+      if (selectedRepo.forks !== null) {
+        const tr3 = createAndAppend('tr', tbody);
+        createAndAppend('td', tr3, { class: 'label', text: 'Forks :' });
+        const fork = createAndAppend('td', tr3, { text: '' });
+        fork.textContent = selectedRepo.forks;
+      }
+      if (selectedRepo.updated_at !== null) {
+        const tr4 = createAndAppend('tr', tbody);
+        createAndAppend('td', tr4, { class: 'label', text: 'Updated :' });
+        const updated = createAndAppend('td', tr4, { text: '' });
+        updated.textContent = new Date(selectedRepo.updated_at).toLocaleString();
```

Comment:
1. Nice use of `toLocaleString()` to format the date according to the user's local computer settings. 
2. You can set the `textContent` directly by using it as the value of the `text` property in the previous line.